webapp-django/crashstats/crashstats/migrations/0001_initial.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-09-25 15:40
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+    ]
```

Comment:
Having an empty initial commit made it muuuuuch easier to go back and forth with the state of the `crashstats` app while working on this PR.